omaha/goopdate/app_manager.cc

```diff
@@ -1431,6 +1434,19 @@ uint32 AppManager::GetDayOfInstall(const GUID& app_guid) const {
   return kUnknownDayOfInstall;
 }
 
+Tristate AppManager::GetAppUsageStatsEnabled(const GUID& app_guid) const {
+  if (!IsAppRegistered(app_guid) && !IsAppUninstalled(app_guid)) {
```

Comment:
I am trying to understand this condition, specifically the !IsAppUninstalled part.

Reply:
So this was following the pattern of GetInstallTimeDiffSec and GetDayOfInstall. Those both check for `!IsAppRegistered && !IsAppUninstalled` and return a sane default in that case. It does seem like a strange case, where we are installed but not registered. I'm not familiar with that scenario, so I thought I'd just follow the existing pattern. Curious your thoughts

E.g.
```cpp
uint32 AppManager::GetInstallTimeDiffSec(const GUID& app_guid) const {
  if (!IsAppRegistered(app_guid) && !IsAppUninstalled(app_guid)) {
    return kInitialInstallTimeDiff;
  }
```

and
```cpp
uint32 AppManager::GetDayOfInstall(const GUID& app_guid) const {
  if (!IsAppRegistered(app_guid) && !IsAppUninstalled(app_guid)) {
    return kInitialDayOfInstall;
  }
```